framework/src/modules/token/cc_method.ts

```diff
@@ -86,37 +86,61 @@ export class TokenInteroperableMethod extends BaseCCMethod {
 	}
 
 	public async beforeCrossChainMessageForwarding(ctx: CrossChainMessageContext): Promise<void> {
-		const { ccm } = ctx;
+		const {
+			transaction: { senderAddress: relayerAddress },
+			ccm,
+		} = ctx;
+		const { ccmID } = getEncodedCCMAndID(ccm);
 		const methodContext = ctx.getMethodContext();
 		const messageFeeTokenID = await this._interopMethod.getMessageFeeTokenID(
```

Comment:
maybe we can move it inside `if (ccm.fee > 0)`

Reply:
do you mean `messageFeeTokenID`? it's used outside of the `if` statement